### PR TITLE
Aligning Apply Buttons

### PR DESCRIPTION
**Pull request by ss1106**
Lined up the apply buttons on the apply page.

### CODE REVIEW

File: src/components/ApplySection.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ function ApplySection(props: any) {
     <article className="applyPage__applicationContent" data-aos="fade-down">
       <li>
         <h3>({props.term})</h3>
-        <h2>{props.role}</h2>
+        <h2>({props.role})</h2>
```
Comment (Niyousha99) [inline src/components/ApplySection.tsx:9]: Any reason the curly braces were removed here but not in the previous line?

**Review by Niyousha99 — CHANGES_REQUESTED** [review]
Can you please apply these changes to aqpply buttons for open applications as well? I believe the css class for those is `applypage__btn`.
image

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,8 +1,87 @@
...
-        "source.fixAll.eslint": true
-    },
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.formatOnSave": true,
+  "editor.formatOnPaste": false,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "files.associations": {
```
Comment (Niyousha99) [inline .vscode/settings.json:8]: What are these added lines for? Can we remove them if they're not necessary?

**Review by Niyousha99 — COMMENTED** [review]
I deleted the c_cpp_properties.json file from your PR btw as I believe it got generated when you were updating your local machine configurations.

File: src/components/ApplySection.tsx (hunk 1 above)
Comment (ss1106) [reply src/components/ApplySection.tsx:9]: I think that was a mistake, do I need to push new changes to revert it?

File: .vscode/settings.json (hunk 2 above)
Comment (ss1106) [reply .vscode/settings.json:8]: I'm not sure where those came from, but I don't think they're necessary.

File: src/components/ApplySection.tsx (hunk 1 above)
Comment (Niyousha99) [reply src/components/ApplySection.tsx:9]: You can push a new change to remove the outer braces, thanks

File: .vscode/settings.json (hunk 2 above)
Comment (Niyousha99) [reply .vscode/settings.json:8]: Ok, please remove the added lines in another commit.

**Review by Niyousha99 — APPROVED** [review]

**Review by Sahitiakella — APPROVED** [review]
LGTM

**Pull request merged**